src/lib/raw_data/xml_reader.cpp

```diff
@@ -532,11 +538,11 @@ std::optional<RawData::RawData> XmlReader::read_mzml(
                 if (tag.value().name == "binaryDataArray") {
                     // precision can be: 64 or 32 (bits).
                     int precision = 0;
+                    // Uncompressed: 0, Zlib compression: 1.
+                    int compressed = -1;
```

Comment:
we should probably use a `bool` type to make it more explicit (true/false for compressed), other than that this is fine.

Reply:
Agree, I have changed it to a bool